sunpy/time/timerange.py

```diff
@@ -25,10 +27,10 @@ class TimeRange(object):
 
     Parameters
     ----------
-    a : str, number, `datetime.datetime`
+    a : str, number, `datetime.datetime`, `astropy.time.Time`
         A time (usually the start time) specified as a parse_time-compatible
         time string, number, or a datetime object.
-    b : str, number, `datetime.datetime`, `datetime.timedelta`, `astropy.units.Quantity` (time)
+    b : str, number, `datetime.datetime`, `datetime.timedelta`, `astropy.units.Quantity` (time),`astropy.time.Time`
```

Comment:
Should the type description be changed to "a time format compatible with parse_time"? And same below.